checkov/sca_package/scanner.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+import json
+import logging
+from collections.abc import Iterable, Sequence
+from pathlib import Path
+from typing import Tuple, Dict, Any
+
+from aiomultiprocess import Pool
+
+from checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning import package_scanning_integration
+from checkov.common.bridgecrew.vulnerability_scanning.package_scanner import TWISTCLI_FILE_NAME, PackageScanner
+
+
+class Scanner:
+    def scan(self, input_output_paths: "Iterable[Tuple[Path, Path]]") -> "Sequence[Dict[str, Any]]":
+        package_scanner = PackageScanner()
+
+        package_scanner.setup_scan()
+
+        scan_results = asyncio.run(
+            self.run_scan_multi(
+                address=package_scanning_integration.get_proxy_address(),
+                bc_api_key=package_scanning_integration.get_bc_api_key(),
+                input_output_paths=input_output_paths,
+            )
+        )
+
+        return scan_results
+
+    async def run_scan_multi(
+        self,
+        address: str,
+        bc_api_key: str,
+        input_output_paths: "Iterable[Tuple[Path, Path]]",
+    ) -> "Sequence[Dict[str, Any]]":
+        args = [
+            (
+                f"./{TWISTCLI_FILE_NAME} coderepo scan --address {address} --token {bc_api_key} --output-file {output_path.absolute()} {input_path.absolute()}",
```

Comment:
Are we still assuming the users have this file? Should we download it for them if the don't?

Reply:
We should wrap the `{input_path.absolute()}` with `"` to handle spaces correctly @gruebel

Reply:
the file is always downloaded